testr/test_helper.py

```diff
@@ -0,0 +1,50 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Provide helper functions that are useful for unit testing.
+"""
+
+import os
+from pathlib import Path
+import socket
+
+__all__ = ['has_sybase', 'on_head_network']
+
+
+def has_sybase():
+    """
+    Return True if the system apparently can run Sybase queries from Python.
+
+    In detail, this is True if the SYBASE and SYBASE_OCS env variables are set
+    and the correct Python shared object library exists on the system.
+    """
+    try:
+        path = Path(os.environ['SYBASE'],
+                    os.environ['SYBASE_OCS'],
+                    'python', 'python34_64r', 'lib', 'sybpydb.so')
+    except KeyError:
```

Comment:
I think on GRETA the keys will be defined in production environments but the dir won't exist.